### ISSUE TITLE
Needlessly verbose (or incorrect?) hover-over string for exported value from node package

### ISSUE THREAD

**Issue by rothfels** [sourcegraph/srclib-javascript#39]
http://cl.ly/35341u1Y2m2Y
Here: https://sourcegraph.com/github.com/expressjs/express@master/-/blob/lib/router/route.js

Looking at the `npm` docs (https://www.npmjs.com/package/array-flatten), I would have expected the type to look more like `array: [...]`. The current type string is quite hard to parse, even for somebody that knows JavaScript.

**Comment by alexsaveliev** [comment]
claimed fixed by #54, requires update of `sourcegraph`
